deepsensor/data/loader.py

```diff
@@ -881,6 +974,9 @@ def task_generation(
             "split" sampling strategy for linked context and target set pairs.
             The remaining observations are used for the target set. Default is
             0.5.
+        patch_size: Sequence[float], optional
+            Desired patch size in lat/lon used for patchwise task generation. Usefule when considering
```

Comment:
There are a few references to lat/lon specifically. Please instead use the DeepSensor standardised coordinate names `x1`/`x2` in comments and variables. The TaskLoader operates only on standardised/normalised data.